src/CloudFtpBridge.Core/Services/FTPSystemActivator.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
```

Comment:
We actually don't need a separate activator for the new file system. The existing FileSystemActivator can activate the new file system for us, we just need to ensure the string added to SupportedFileSystems is the full type name of the new file system.

Reply:
This has been resolved in the latest commit to AddNewFTPOption branch.